e2e/src/usecases/Settings.js

```diff
@@ -23,11 +23,40 @@ export default Settings = () => {
     await element(by.id('ProfileEditName')).clearText()
     await element(by.id('ProfileEditName')).replaceText(`${randomName}`)
     await element(by.id('SaveButton')).tap()
-    await expect(element(by.text('Your name and picture were saved successfully.'))).toBeVisible()
+    await waitFor(element(by.text('Your name and picture were saved successfully.')))
+      .toBeVisible()
+      .withTimeout(1000 * 10)
```

Comment:
Why such a long timeout? 
If I remember correctly it only saves this locally now, so there shouldn't be any network delay.

Reply:
The `waitFor` is a maximum of 10 seconds I was running into some issues where the expect was firing too soon before the banner was displayed so opted for this instead.